Polyhedron/demo/Polyhedron/Plugins/Mesh_3/Io_image_plugin.cpp

```diff
@@ -222,7 +222,7 @@ class Io_image_plugin :
       connect(planeSwitch, SIGNAL(triggered()),
               this, SLOT(selectPlanes()));
       Scene* true_scene = dynamic_cast<Scene*>(scene);
-      connect(true_scene,SIGNAL(selectionChanged(int)),
+      connect(true_scene,SIGNAL(itemIndexSelected(int)),
```

Comment:
The two functions emit a different signal. One is used in case a plugin or an item must do something when the selection has changed, the other is only used to update the TreeView. They are not always called in the same conditions.